phpunit/navigation-test.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * This class is supposed to test the functionality of the navigation.php
```

Comment:
`is supposed to test` -> `tests`,
`of the navigation.php` -> `of block navigation `

Reply:
Fixed in 38dbd8e19b65218ccebf46c07ef8515dc8ac667a